### DISCUSSION TITLE
Version showing as snapshot

### DISCUSSION THREAD

**Discussion by naveen8384** [tchiotludo/akhq#845] [Q&A] [answered]
After updating to latest version 0.18.0 version is showing as **Snapshot**, but in old versions version number it displayed. Is there any way to show version number instead of Snapshot

![image](https://user-images.githubusercontent.com/4356276/136343038-203c997c-33e4-4f23-9d4e-88ff1052ab92.png)

![image](https://user-images.githubusercontent.com/4356276/136342040-64778676-51d7-4348-a6a6-097788d9fb59.png)

**Comment by n-i-c-o-l-a-s** [comment]
Also interested.

**Comment by tchiotludo** [comment, marked as answer]
Issue is here : 
- https://github.com/tchiotludo/akhq/issues/861
Ready to be pickup :smile:
